docs/sources/features/datasources/cloudmonitoring.md

```diff
@@ -335,3 +335,22 @@ Only available in Grafana v7.1+.
 
 Click on a time series in the panel to see a context menu with a link to View in Metrics Explorer in Google Cloud Console. Clicking that link opens the Metrics Explorer in the Google Cloud Console and runs the query from the Grafana panel there.
 The link navigates the user first to the Google Account Chooser and after successfully selecting an account, the user is redirected to the Metrics Explorer. The provided link is valid for any account, but it only displays the query if your account has access to the GCP project specified in the query.
+
+## Out-of-the-box dashboards
+
+> Only available in Grafana v7.3+.
+
+The updated Cloud Monitoring data source ships with pre-configured dashboards for five of the most popular GCP services:
+
+- BigQuery Query Monitoring,
```

Comment:
Remove comma after bullets. No period after the last bullet.

Reply:
Also, the topic should have lazy numbering for all procedure sections. Need correction.

Reply:
what do you mean by "lazy numbering". Can you be more specific or point me to an example? thanks!

Reply:
All numbering should be 1, and not 1,2,3, ... they will render correctly in your preview.

Here is an explanation:

Use lazy numbering for long lists
Markdown is smart enough to let the resulting HTML render your numbered lists correctly. For longer lists that may change, especially long nested lists, use “lazy” numbering:

1.  Foo.
1.  Bar.
    1.  Foofoo.
    1.  Barbar.
1.  Baz.